build.gradle.kts

```diff
@@ -30,6 +30,7 @@ fun log4j(
     "org.apache.logging.log4j:log4j-$module:2.10.0"
 }
 
-val wrapper = tasks["wrapper"] as Wrapper
-wrapper.gradleVersion = "4.9"
-wrapper.distributionType = Wrapper.DistributionType.ALL
+tasks.getByName("wrapper", Wrapper::class).apply {
```

Comment:
```suggestion
tasks.wrapper {
```

Reply:
The current syntax seems to be more universal.